src/styles/variables.scss

```diff
@@ -0,0 +1,7 @@
+$watch-size: 80vmin;
+$sec-size: 38vmin;
+$min-size: 20vmin;
+$center-radius: 5vmin;
+$basic-time: 10s;
```

Comment:
It seems like `$basic-time` is used for animation duration. According to code style rules, it's better to suffix the variable with its purpose, like `$animation-duration-basic` to make its usage more explicit.